src/cmd/maturity.go

```diff
@@ -84,3 +80,39 @@ func GetValues(s *opslevel.ServiceMaturity, fields ...string) []string {
 	}
 	return output
 }
+
+func writeOutput(client *opslevel.Client, data []opslevel.ServiceMaturity) {
```

Comment:
This is 😍  i've been meaning to do something like this in a more generic way but this is good because it starts to form the pattern.